app/config/errors.php

```diff
@@ -364,6 +364,28 @@
         'code' => 416,
     ],
 
+    /** VCS */
+    Exception::INSTALLATION_NOT_FOUND => [
+        'name' => Exception::INSTALLATION_NOT_FOUND,
+        'description' => 'Installation with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::PROVIDER_REPOSITORY_NOT_FOUND => [
+        'name' => Exception::PROVIDER_REPOSITORY_NOT_FOUND,
+        'description' => 'Repository with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::REPOSITORY_NOT_FOUND => [
+        'name' => Exception::REPOSITORY_NOT_FOUND,
+        'description' => 'VCS Repository with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::PROVIDER_CONTRIBUTION_CONFLICT => [
+        'name' => Exception::PROVIDER_CONTRIBUTION_CONFLICT,
+        'description' => 'External ontribution is already authorized.',
+        'code' => 409,
+    ],
+
```

Comment:
```suggestion
    Exception::PROVIDER_CONTRIBUTION_CONFLICT => [
        'name' => Exception::PROVIDER_CONTRIBUTION_CONFLICT,
        'description' => 'External contribution is already authorized.',
        'code' => 409,
    ],
```
Not sure what this error means as well 🤔  we need to make the type and message more clear

Reply:
There is "Authorize" button on PR when contribution is external (from fork). We don't automatically build those for security reasons.

When you click the button, you go to console where you have "Authorize" button. If you authorize it, then refresh page, and try to authorize again, this error will be used. It means you already authorized that contribution and is considered trusted.

We use word CONFLICT for exception name to follow HTTP standards